schema_based_on_diagram.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE medical_database;
+
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly create a "join table" for the many-to-many relationships even if they are not shown on the diagram. You may create a table between `medical_histories` and `treatments`.
- [ ] Also you may set foreign keys to this table and create indexes for these foreign keys.

Reply:
DONE!